test/setup_environment.sh

```diff
@@ -127,6 +127,8 @@ if [[ -n ${build_coldcard} ]]; then
         fi
     fi
     # Apply patch to make simulator work in linux environments
+    git config --global user.email "you@example.com"
```

Comment:
probably a better way somehow but it didn't like that I had nothing configured locally on docker

Reply:
It's sufficient to have an environment variable named `EMAIL` set to the email. You can add that to the dockerfile.

Reply:
done, and added some more notes to tell someone how to run it locally